lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -132,12 +142,25 @@ class RenderForm extends React.Component {
   renderSupportForm() {
     let userElement;
     if (this.props.context.user) {
+      if (this.state.details){
       userElement = (<LoggedInUser
         userInformation={this.props.context.user}
         submitFormUrl={this.props.context.submitFormUrl}
         setErrorState={this.setErrorState}
+        setDisplay={this.setDisplay}
         submitForm={this.submitForm}
+        showDisplay=true
       />);
+    }else{
```

Comment:
Instead of copying almost the entire component with the same props, I would do the conditional only on the prop itself like this: 
`showDisplay={this.state.details ? true :  false}`

Reply:
good idea